doc/source/getting-started-mac-tutorial.rst

```diff
@@ -0,0 +1,155 @@
+
+
+.. code-block:: shell-session
+
+    $ python3 vol.py -f contact_me banners.Banners
+        
+        Volatility 3 Framework 2.4.2
+
+        Progress:  100.00               PDB scanning finished
+        Offset  Banner
+        
+        0x4d2c7d0       Darwin Kernel Version 16.7.0: Thu Jun 15 17:36:27 PDT 2017; root:xnu-3789.70.16~2/RELEASE_X86_64
+        0xb42b180       Darwin Kernel Version 16.7.0: Thu Jun 15 17:36:27 PDT 2017; root:xnu-3789.70.16~2/RELEASE_X86_64
+        0xcda9100       Darwin Kernel Version 16.7.0: Thu Jun 15 17:36:27 PDT 2017; root:xnu-3789.70.16~2/RELEASE_X86_64
+        0x1275e7d0      Darwin Kernel Version 16.7.0: Thu Jun 15 17:36:27 PDT 2017; root:xnu-3789.70.16~2/RELEASE_X86_64
+        0x1284fba4      Darwin Kernel Version 16.7.0: Thu Jun 15 17:36:27 PDT 2017; root:xnu-3789.70.16~2/RELEASE_X86_64
+        0x34ad0180      Darwin Kernel Version 16.7.0: Thu Jun 15 17:36:27 PDT 2017; root:xnu-3789.70.16~2/RELEASE_X86_64
+        
+
+The above command helps us to find the memory dump's Darwin kernel version. Now using the above banner we can search for the needed ISF file.
+If ISF file cannot be found then, follow the instructions on :ref:`getting-started-macos-tutorial:Procedure to create symbol tables for macOS`. After that, place the ISF file under the ``volatility3/symbols/mac`` directory.
+
+mac.pslist
+~~~~~~~~~~~~
+
+.. code-block:: shell-session
+
+    $ python3 vol.py -f contact_me mac.pslist.PsList
+
+        Volatility 3 Framework 2.4.2
+        Progress:  100.00               Stacking attempts finished
+
+        PID     PPID    COMM
+
+        0       0       kernel_task
+        1       0       launchd
+        35      1       UserEventAgent
+        38      1       kextd
+        39      1       fseventsd
+        37      1       uninstalld
+        45      1       configd
+        46      1       powerd
+        52      1       logd
+        58      1       warmd
+        .....
+
+``mac.pslist`` helps us to list the processes which are running, their PIDs and PPIDs.
+
+mac.pstree
+~~~~~~~~~~~~
+
+.. code-block:: shell-session
+
+    $ python3 vol.py -f contact_me mac.pstree.PsTree
+        Volatility 3 Framework 2.4.2
+        Progress:  100.00               Stacking attempts finished
+        PID     PPID    COMM
+
+        35      1       UserEventAgent
+        38      1       kextd
+        39      1       fseventsd
+        37      1       uninstalld
+        204     1       softwareupdated
+        * 449   204     SoftwareUpdateCo
+        337     1       system_installd
+        * 455   337     update_dyld_shar
+
+``mac.pstree`` helps us to display the parent child relationships between processes.
+
+mac.ifconfig
+~~~~~~~~~~
+
+we can use the ``mac.ifconfig`` plugin to get information about the configuration of the network interfaces of the host under investigation.
```

Comment:
Unlike other descriptions, this is above the results of the tool execution. Is there a reason for this?

Reply:
You're right. It seems to be a mistake. I've moved the location.